src/common/transformations/src/transformations/op_conversions/convert_interpolate11_downgrade.cpp

```diff
@@ -42,15 +43,25 @@ ov::pass::ConvertInterpolate11ToInterpolate4::ConvertInterpolate11ToInterpolate4
         if (interpolate_v11->get_attrs().shape_calculation_mode ==
             ov::op::util::InterpolateBase::ShapeCalcMode::SCALES) {
             v4_input_scales = interpolate_v11->input_value(1);
-            v4_input_output_shape = opset4::Constant::create(element::i32, Shape{1}, {1});
+            v4_input_output_shape = opset4::Constant::create(element::i32, Shape{}, {1});
+            if (with_axes_input) {
+                v4_input_output_shape = std::make_shared<opset4::Broadcast>(
+                    v4_input_output_shape,
+                    std::make_shared<opset4::ShapeOf>(interpolate_v11->input_value(2)));
+            }
```

Comment:
Is there any reason to not use always available shape of v11 "scale_size_input" (`interpolate_v11->input_value(1)`) instead of optional "axes" shape?

Also the runtime info should be copied to all of the subgraph ops (ShapeOf as well).
```suggestion
v4_input_output_shape = std::make_shared<opset4::Broadcast>(
opset4::Constant::create(element::i32, Shape{}, {1}),
std::make_shared<opset4::ShapeOf>(interpolate_v11->input_value(1)));
```

Reply:
I can broadcast to the `scales_or_sizes` input instead of `axes` only if it's guaranteed that their shapes will always be equal so that the GPU plugin assert doesn't trigger an exception. Otherwise I'd rather use `axes` input's shape to solve a particular error that the recent change is trying to address.

Reply:
As we discussed, in my opinion the GPU assertions should be relaxed and adjusted to validate only the input related to the `Interpolate::ShapeCalcMode` (sizes or scales, not both).

But ok I see the point, so to fix the issue without touching the GPU code, looks like the axes shape needs to be the target shape. And in case of default axes, the dummy input should be broadcasted to the rank of data shape. 
Otherwise the following GPU assertion will complain, as it will be compared to the axes size aligned with rank of the input shape:
https://github.com/openvinotoolkit/openvino/blob/7442a17240bb1d3b9146527ce215919333c587d9/src/plugins/intel_gpu/src/plugin/ops/interpolate.cpp#L48 
The axes calculation:
https://github.com/openvinotoolkit/openvino/blob/7442a17240bb1d3b9146527ce215919333c587d9/src/plugins/intel_gpu/src/plugin/ops/interpolate.cpp#L35-L44